<launch>
  
  <!-- Start the VRPN Client (only needed if recording data)-->
<!--  <include file="/opt/ros/kinetic/share/vrpn_client_ros/launch/SO_vicon.launch"/> -->

  <!-- Start the camera node -->
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="320" />
    <param name="image_height" value="240" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
    <param name="framerate" value="60"/>
  </node>
  
  <!-- Start the optic flow node -->
  <node pkg="object_avoidance" type="Flow6060.py" name="Flow_node" output="screen" />

</launch>
